use-cases/financial_payment_classification/financial payment classification.ipynb

```diff
@@ -0,0 +1,1849 @@
+{
```

Comment:
<p>If you want to include any figures, please use markdown tables or separate jpg images, rather than cell outputs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3017/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>